packages/core/admin/admin/src/content-manager/components/ContentTypeFormWrapper.tsx

```diff
@@ -88,7 +88,13 @@ const ContentTypeFormWrapper = ({
   const { setCurrentStep } = useGuidedTour();
   const { trackUsage } = useTracking();
   const { push, replace } = useHistory();
-  const [{ query, rawQuery }] = useQueryParams();
+  const [{ query, rawQuery }] = useQueryParams<{
+    plugins?: {
+      i18n?: {
+        locale?: string;
+      };
+    };
```

Comment:
didn't we only sent the locale in v4 , without the plugins[i18n][locale]?

Reply:
This is the types for how we extract it from the URL in the FE. When we actually make the delete request it isn't nested like this